indico/modules/events/sessions/clone.py

```diff
@@ -51,30 +58,43 @@ def _clone_sessions(self, new_event):
                           subqueryload('acl_entries')))
         for old_sess in query:
             sess = Session()
-            sess.populate_from_attrs(old_sess, attrs)
-            sess.blocks = list(self._clone_session_blocks(old_sess.blocks))
+            if event_exists:
+                sess.populate_from_attrs(old_sess, attrs, skip='friendly_id')
```

Comment:
This doesn't work, `skip` is supposed to be a collection... Anyway, you do not need to skip since you overwrite it in the next line anyway. I would add something like this before the loop though since it's cleaner:

```python
if event_exists:
    attrs.discard('friendly_id')
```